### PR TITLE
Update LTS infos

### PR DESCRIPTION
**Pull request by fkaminski**
See #981

### CODE REVIEW

File: docs/manual/installation/update-contao.en.md [hunk 1]
```diff
@@ -39,9 +39,9 @@ when writing these lines is **version 4.10.4**.
 
 ### Long-Term Support Versions
 
-With version 2.11, the [release cycle of Contao](https://contao.org/de/release-plan.html) has been adjusted and 
-Long-Term Support (LTS) versions have been introduced that are supported and updated for 24 months, even if newer Contao 
-versions have been released in the meantime. An overview of all Contao versions is available on [Wikipedia](https://en.wikipedia.org/wiki/Contao#Versions).
+The [release cycle](https://contao.org/de/release-plan.html) of Contao also includes a Long-Term Support. Contao LTS versions are 
+supported and updated for up to 4 years, even if newer Contao versions have been released in the meantime. An overview of all Contao versions 
+can be found on [Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
```
Comment (fritzmg) [inline docs/manual/installation/update-contao.en.md]: May be we could mention that the LTS gets a total 3 years of active support and 1 year of security only support (no non-security related bug fixes).

File: docs/manual/installation/update-contao.de.md [hunk 2]
```diff
@@ -40,11 +40,10 @@ Die aktuelle Bugfix-Version von Contao ist beim Schreiben dieser Zeilen die **Ve
 
 ### Long-Term-Support-Versionen
 
-Mit Version 2.11 wurde der [Release-Zyklus von Contao](https://contao.org/de/release-plan.html) angepasst und 
-[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support)-Versionen (LTS) 
-eingeführt, die 24 Monate lang unterstützt und mit Updates versorgt werden, auch wenn zwischenzeitlich schon neuere 
-Contao-Versionen veröffentlicht wurden. Eine Übersicht aller Contao Versionen gibt es auf 
-[Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
+Der [Release-Zyklus](https://contao.org/de/release-plan.html) von Contao beinhaltet auch den
```
Comment (netzarbeiter) [inline docs/manual/installation/update-contao.de.md]:
```suggestion
Der [Release-Zyklus](https://contao.org/de/release-plan.html) von Contao beinhaltet auch Versionen mit 
```

File: docs/manual/installation/update-contao.de.md [hunk 3]
```diff
@@ -40,11 +40,10 @@ Die aktuelle Bugfix-Version von Contao ist beim Schreiben dieser Zeilen die **Ve
 
 ### Long-Term-Support-Versionen
 
-Mit Version 2.11 wurde der [Release-Zyklus von Contao](https://contao.org/de/release-plan.html) angepasst und 
-[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support)-Versionen (LTS) 
-eingeführt, die 24 Monate lang unterstützt und mit Updates versorgt werden, auch wenn zwischenzeitlich schon neuere 
-Contao-Versionen veröffentlicht wurden. Eine Übersicht aller Contao Versionen gibt es auf 
-[Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
+Der [Release-Zyklus](https://contao.org/de/release-plan.html) von Contao beinhaltet auch den 
+[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support). Die Contao LTS-Versionen werden bis zu 4 Jahre
```
Comment (netzarbeiter) [inline docs/manual/installation/update-contao.de.md]:
```suggestion
[Long Term Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support). Die Contao-Versionen mit verlängertem Supportzeitraum werden während 3 Jahren mit Bugfixes und 1 Jahr mit  sicherheitsrelevante Updates
```

File: docs/manual/installation/update-contao.de.md [hunk 4]
```diff
@@ -40,11 +40,10 @@ Die aktuelle Bugfix-Version von Contao ist beim Schreiben dieser Zeilen die **Ve
...
 ### Long-Term-Support-Versionen
 
-Mit Version 2.11 wurde der [Release-Zyklus von Contao](https://contao.org/de/release-plan.html) angepasst und 
-[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support)-Versionen (LTS) 
-eingeführt, die 24 Monate lang unterstützt und mit Updates versorgt werden, auch wenn zwischenzeitlich schon neuere 
-Contao-Versionen veröffentlicht wurden. Eine Übersicht aller Contao Versionen gibt es auf 
-[Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
+Der [Release-Zyklus](https://contao.org/de/release-plan.html) von Contao beinhaltet auch den 
+[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support). Die Contao LTS-Versionen werden bis zu 4 Jahre 
+lang unterstützt und mit Updates versorgt, auch wenn zwischenzeitlich schon neuere Contao-Versionen veröffentlicht wurden. Eine Übersicht
```
Comment (netzarbeiter) [inline docs/manual/installation/update-contao.de.md]:
```suggestion
versorgt, auch wenn zwischenzeitlich schon neuere Contao-Versionen veröffentlicht wurden. Eine Übersicht 
```

File: docs/manual/installation/update-contao.de.md [hunk 5]
```diff
@@ -40,11 +40,10 @@ Die aktuelle Bugfix-Version von Contao ist beim Schreiben dieser Zeilen die **Ve
...
 
-Mit Version 2.11 wurde der [Release-Zyklus von Contao](https://contao.org/de/release-plan.html) angepasst und 
-[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support)-Versionen (LTS) 
-eingeführt, die 24 Monate lang unterstützt und mit Updates versorgt werden, auch wenn zwischenzeitlich schon neuere 
-Contao-Versionen veröffentlicht wurden. Eine Übersicht aller Contao Versionen gibt es auf 
-[Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
+Der [Release-Zyklus](https://contao.org/de/release-plan.html) von Contao beinhaltet auch den 
+[Long-Term-Support](https://de.wikipedia.org/wiki/Support_(Dienstleistung)#Long_Term_Support). Die Contao LTS-Versionen werden bis zu 4 Jahre 
+lang unterstützt und mit Updates versorgt, auch wenn zwischenzeitlich schon neuere Contao-Versionen veröffentlicht wurden. Eine Übersicht 
+aller Contao Versionen gibt es auf [Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
```
Comment (netzarbeiter) [inline docs/manual/installation/update-contao.de.md]:
```suggestion
aller Contao-Versionen gibt es auf [Wikipedia](https://de.wikipedia.org/wiki/Contao#Versionen).
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko.

File: docs/manual/installation/update-contao.en.md (hunk 1 above)
Comment (fkaminski) [reply docs/manual/installation/update-contao.en.md]: See Bjarke review.

**Review by fritzmg — APPROVED** [review]

**Comment by netzarbeiter** [comment]
If the text is ok for you, I would also change the Wikipedia entry.

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**